### ISSUE TITLE
VPN reconnection error.  Adaway latest version.

### ISSUE THREAD

**Issue by faizaff** [AdAway/AdAway#2373]
Yesterday I installed for the first time, and worked fluently, I was so happy to get a relief from Adgaurd which I have still in my system.
But today I noticed the vpn icon missing on my status bar just to verify I checked the notification s and there it was, "Ad-Blocker Vpn reconnection error" please help me out how I get rid of this error, and remain connected. Restarting of the device did not help, refreshing and re-downloading of host files etc did not work.
Please help me out with the solution.

**To Reproduce**

**Expected behavior**

**Screenshots**

**Device** Realme C3
 - AdAway version: [e.g. 
5.1.]
 - Device: RMX-2027
 - Android version:  Android 10
 - Non-Rooted
Thankyou,
Ashfaq Shaikh

**Additional context**
Add any other context like logcat about the problem here. Use adb logcat if you have developer settings enabled on your device or use any application like [MatLog](https://play.google.com/store/apps/details?id=com.pluscubed.matlog) to save logs. Include the crash report in the issue in a code block, a file attachment, or put it in a gist and provide link to that gist.

**Comment by PerfectSlayer** [comment]
Hello,

It looks like the VPN service was killed if the VPN icon is missing.
Have you tried the round button at the bottom center of the app? It can pause / resume the ad-blocker which should restart the VPN service.

Regards

**Comment by faizaff** [comment]
Yes I did, a number of times, but vpn fails to reconnect ! And yes Adgaurd
when pressed into service, or any other vpn like Max or touch vpn,
immediately brings the icon in the statu bar showing connected.
I reinstalled the adaway now from F-Droid but no change.
Help me out please.
Thanking you in advance.
Ashfaq Shaikh

**Comment by faizaff** [comment]
Just noticed, when IDM OR ADM Services are on ADAWAY VPN Fails, Even
Telegram when opened, vpn fails to hold on
VPN Connection, needs to be strengthened, than it can kill all the
competition to become numero 1
Hope you will check and confirm. Waiting for the tweak.
Ashfaq Shaikh

**Comment by faizaff** [comment]
See how dns66 is working!

**Comment by zgfg** [comment]
See also #2154

**Comment by balaji345** [comment]
I was having the same problem when I unlocked the bootloader on my Redmi note 9 Merlin and tried installing the latest stable. I then downloaded and installed the preview version 5.0.10-201010 . This is working fine. For some reason the stable version kept giving the error you are getting. There appears no solution on the web for this when I searched. Hope the bug is addressed in the future release

**Comment by faizaff** [comment]
Where to get this version link please !

**Comment by balaji345** [comment]
https://app.adaway.org/beta.apk

**Comment by faizaff** [comment]
Thanks, got it now.
